app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
+        return
+    if len(route) > 1:
+        route.pop()
+    else:
+        new_file_name = route[0]
+        with (
+            open(command[1], "r") as file_in,
+            open(new_file_name, "x") as file_out,
+        ):
+            file_out.write(file_in.read())
+        os.remove(command[1])
+        return
+    for directory in route:
+        if not os.path.exists(directory):
+            os.mkdir(directory)
+        os.chdir(directory)
```

Comment:
Code Style: You are using 'os.chdir' to change the current working directory. This can be problematic because it changes the state of your program. It would be better to use 'os.path.join' to concatenate parts of the path. This way, your program remains stateless and is more predictable.